### DISCUSSION TITLE
feature request | Analog Trigger

### DISCUSSION THREAD

**Discussion by halop** [cathery/sys-con#39] [Feature requests]
Joycons and Pro controllers no longer have analog triggers. But if you connect a Gamecube controller to the switch, you can use the analog trigger in some games, e.g. Trials Rising & GRID Autosport.
Is it possible to use the analog trigger function on an Xbox360 controller, and/or to emulate a Gamecube controller?

**Comment by cathery** [comment]
How does it work, you accelerate faster if the harder you press the trigger? If so, I don't see any variables I could use to pass that info. Maybe it uses some unused memory, I'll look into it

**Comment by halop** [comment]
Yes, and as far as I know it is only officially supported by a Gamecube controller.

**Comment by cathery** [comment]
So there's a section in HID shared memory which specifically holds states of all gamecube controller triggers. The thing is, that section is read-only. But someone told me there's a very slow way to write to read-only chunks of memory. So maybe it's possible?

**Comment by goooooouwa** [comment]
It would be great if we can use controllers with analog triggers on games support it like Grid Autosports. Looking forward to a solution on this. :)
